Doc/whatsnew/3.14.rst

```diff
@@ -210,6 +210,10 @@ configuration mechanisms).
 
 Other language changes
 ======================
+* Constant comparsion expressions are now folded and evaluated before runtime.
```

Comment:
I don't know how we are actually ordering the changes. Since this is a fairly niche optimization, I would put it at the end of the list.

Reply:
Usually, new changes are added at the end of a section.